docs/src/development/local/_index.md

```diff
@@ -1,151 +1,92 @@
+    ```
+
+You can now access your code from the project directory on your computer.
+The CLI created a `.platform/local` directory that's excluded from Git. 
+It contains builds and local metadata about your project.
 
 You can now make changes to your project without pushing to Platform.sh each time to test them. 
 Instead, you can locally build your application using the Platform.sh CLI.
 
 Note that if your app contains services, you need to open an SSH tunnel to connect to them.
 For more information, see how to [connect services](../../add-services#2-connect-the-service).
 
-## 2. Build your site locally
+## 2. Connect to services
+
+If your app requires services to run, you have two options for developing locally:
+
+- [Tethered local development](./tethered.md) involves running your app on a local web server
+  but keeping all other services on Platform.sh and connecting to them over an SSH tunnel.
+- [Untethered local development](./untethered.md) involves running your entire site locally,
+  including all services.
+
+Choose the option that works for you and get your services running.
 
-Before you build your site, 
-check that any runtime or tools needed for the build process are available 
-in your local environment.
+## 3. Build your site locally
 
 If you want your local development environment to be enclosed 
 so your main system remains unaffected by the build and its dependencies, 
 you can use a local virtual machine.
 
 To build your site locally:
 
-1. Run the following command:
+1.  Install any dependencies or tools needed for the build process.
 
-   ```bash
-   platform build
-   ```
+2.  Run the following command:
 
-2. When prompted, specify the source directory and the build destination.
+    ```bash
+    platform build
+    ```
 
-   Your app is built locally.
-   At the root of your local project directory, a `_www` subdirectory is created.
-   It is a symbolic link pointing to the currently active build located in the `.platform/local/builds` directory.
+    Your app is built in the `.platform/local/builds` directory.
+    Unless you specify otherwise with the `--destination` flag,
+    a symbolic link from your build is created as the `_www` directory at your project root.
 
-3. To check that the build was successful:
-   - If you are familiar with how to set up a local web server or already have one, 
-     you can move to the `_www` directory to find your build
-     and run your web server and services from there.
-   - If you don’t want to set up everything manually, 
-     you can switch to a Docker-based local development environment tool 
-     such as [DDEV](./ddev.md), [Docksal](./docksal.md) or [Lando](./lando.md).
-     
-     Other options include [tethered local development](./tethered.md) and [untethered local development](./untethered.md).
+3.  To check that the build was successful, move to the `_www` directory
+    and run a web server.
```

Comment:
If this is about DDEV, this is misdirection. I may be missing context.

Reply:
As above, this part shouldn't be about DDEV.